src/jit-format/jit-format.cs

```diff
@@ -464,7 +464,15 @@ public static int Main(string[] args)
             {
                 if (!RunClangFormat(filenames, config.Fix, verbose))
                 {
-                    Console.WriteLine("Clang-Format needs to be rerun in fix mode");
+                    Console.WriteLine("Clang-format found formatting errors. Run:");
+                    if (returncode == -1)
```

Comment:
The analogous tidy case above seems to need a fix as well.  Also, you have to run without -noformat to get here, right?  I think that you need to move this error handling out to the main path and handle the various combinations of error conditions, especially if you're going to tailor the suggested command line with -untidy/-noformat based on the situation.